### PR TITLE
v3.7.5 release

### PR DESCRIPTION
**Pull request by moderakh**
v3.7.5 release.

### CODE REVIEW

File: sdk/cosmos/changelog/README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ## Changelog
```
Comment (weshaggard) [inline sdk/cosmos/changelog/README.md:1]: Why isn't this file called CHANGELOG.md?

Comment (kushagraThapar) [reply sdk/cosmos/changelog/README.md:1]: Old V3 stuff

Comment (kushagraThapar) [reply sdk/cosmos/changelog/README.md:1]: We can update it later. Though we have tried not to change V3 structure at this point - just want to leave it as it is as we rarely interact with it.

Comment (weshaggard) [reply sdk/cosmos/changelog/README.md:1]: OK, I missed this was in the v3 branch.

**Pull request merged**